usr/share/rear/rescue/GNU/Linux/310_network_devices.sh

```diff
@@ -107,17 +120,6 @@ for network_interface in $network_interfaces ; do
     # (until someone implements a better solution that works on all bash 3.x versions):
     if [ -d "/sys/class/net/$network_interface/bridge" ]; then
         # Consider bridges linked to physical interfaces only (directly or not)
-        function is_linked_to_physical () {
-            local devpath=$1
-            # If device is a physical device, return success
-            [ ! -e $devpath/device ] || return 0
-            # Otherwise recurse on lower devices (if any)
-            for dev in $(/bin/readlink -f $devpath/lower_* 2>/dev/null); do
-                ! is_linked_to_physical $dev || return 0
-            done
-            # Device is not a physical device
-            return 1
-        }
         is_linked_to_physical /sys/class/net/$network_interface || network_interface=""
```

Comment:
What a pity!
I liked so much your "very local" function
definition plus usage (and deletion/unset)
all within the smallest possible scope.

Reply:
Well, it's unoptimized. Since the function can be used for other reasons, i moved it globally.

Reply:
Don't worry,
my comment was not meant as a request to change it back.
Ultimately it is your code and you make the decisions how
you implement it (unless you really break something in ReaR).